cypress/integration/linkEmbedder.spec.ts

```diff
@@ -15,24 +15,38 @@ describe('Link gets replaced with embedding: ', () => {
       .find('.one-click-embedding.gist-frame')
       .click()
     cy.getMarkdownBody()
-      .find('iframe')
-      .should('have.attr', 'title', 'gist schacon/1')
+      .find('iframe[data-cy=gh-gist]')
+      .should('be.visible')
   })
 
   it('YouTube', () => {
     cy.codemirrorFill('https://www.youtube.com/watch?v=YE7VzlLtp-4')
     cy.getMarkdownBody()
-      .find('.one-click-embedding.embed-responsive-item')
+      .find('.one-click-embedding-preview')
+      .should('have.attr', 'src', 'https://i.ytimg.com/vi/YE7VzlLtp-4/maxresdefault.jpg')
+      .parent()
```

Comment:
I think we should find a way to test the click-shield-component itself instead of using a specific implementation. But I don't know how. Maybe there are unit tests for React? :thinking:  Can you add a todo here for this?

Reply:
Sounds almost like [storybook tests](https://storybook.js.org/docs/react/workflows/testing-with-storybook).

I added a TODO entry.